projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -289,17 +289,19 @@ public function page_admin_scripts() {
 			// Required for Analytics.
 			wp_enqueue_script( 'jp-tracks', '//stats.wp.com/w.js', array(), gmdate( 'YW' ), true );
 		}
+		// The script handle for the initial state.
+		$initial_state_handle = 'jetpack-publicize';
```

Comment:
I suspect the changes here will break everything that's not jetpack-publicize. Or if they don't, it's just by luck that whatever you're doing here happens to also load before the `'react-plugin'` script tries to do anything with it.

Reply:
Yeah, I understand it's not very intuitive, but since our initial state logic is a mess, as mentioned before (pdWQjU-KI-p2). If we had the initial state consolidated, it would be rendered independent of any script handle as early as possible to let any script use it.

Why this works here is that `'react-plugin'` always has a dependency named `jetpack-publicize` via the dependency extraction webpack plugin and that gets rendered first and that is why it works for `react-plugin` as well.

Reply:
> Why this works here is that `'react-plugin'` always has a dependency named `jetpack-publicize`

For now. That seems super fragile.

Reply:
Should we associate that with `jetpack-connection` now as we have unbundled that as well?